### PR TITLE
Add/eslint plugin mdx

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

Adds linting for MDX files

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] `yarn lint`

### CODE REVIEW

File: apps/docs/pages/availability.mdx [hunk 1]
```diff
@@ -8,8 +8,7 @@ title: Availability
 
 1. Go to [Your Availability](https://app.cal.com/availability) page within your cal.com account.
```
Comment (hariombalhara) [inline apps/docs/pages/availability.mdx:9]: @zomars I don't think we even use apps/docs now. Maybe we can delete the docs folder?

Comment (hariombalhara) [reply apps/docs/pages/availability.mdx:9]: Cool

**Comment by hariombalhara** [comment]
There are conflicts in it, I think we can quickly merge after that,

**Review by PeerRich — CHANGES_REQUESTED** [review]
merge conflicts. moved to draft

**Comment by zomars** [comment]
Closing for now. MDX linting is not a priority right now.

**Pull request closed without merging**